database/migrations/2023_11_22_121149_create_values_table.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('attributes', function (Blueprint $table) {
+            $table->dropColumn('model');
+        });
+
+        Schema::create('values', function (Blueprint $table) {
+            $table->char('id', 36)->collation('ascii_bin')->primary();
+
+            $table->char('attribute_id', 36)->collation('ascii_bin');
+            $table->char('object_id', 36)->collation('ascii_bin');
+
+            $table->char('position_id', 36)->collation('ascii_bin')->nullable();
+            $table->char('ref_object_id', 36)->collation('ascii_bin')->nullable();
+            $table->string('string_value')->nullable();
+            $table->integer('int_value')->nullable();
+            $table->dateTime('datetime_value')->nullable();
```

Comment:
A date to nie będzie?

Reply:
tak jak bool będzie przechowany w kolumnie int, tak date w kolumnie datetime (chyba że się okaże, że warto inaczej)